core/src/main/java/com/kin/ecosystem/core/data/internal/Environment.java

```diff
@@ -13,7 +13,7 @@
 		"GDF42M3IPERQCBLWFEZKQRK77JQ65SCKTU3CW36HZVCX7XX5A5QXZIVK",
 		"https://horizon.kinfederation.com",
 		"Kin Mainnet ; December 2018",
-		"TODO",
+		"https://migration-service.kinmarketplace.com/migrate?address=",
```

Comment:
why do we set here the api params? `address`

Reply:
@drEfrati ^^^

Reply:
because this is the only url as far as I know, no other options

Reply:
unfortunately this is part of the migration module, we're keeping it the way it is